website/client/components/inventory/equipment/equipGearModal.vue

```diff
@@ -59,11 +63,45 @@
       width: 282px;
     }
 
-    .bordered {
+    .classTag {
+      height: 24px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    .className {
+      height: 24px;
+      font-family: Roboto;
+      font-size: 16px;
+      font-weight: bold;
+      font-stretch: condensed;
```

Comment:
this doesn't work, you have to use the Roboto Condensed filename, also isn't an already existing class for this? that at least doesn't require re-declaring all these typhography rules

Reply:
yeah probably too much copied from zeplin ^^